README.md

```diff
@@ -50,3 +48,9 @@ may not work properly on specific/all platforms.
 - ATMEL AVR Series (incl. Arduino): avr-gcc
 - x86, x86_64: gcc, clang
 
+Contributors
+------------
+
+- eflukx
+- hasegaw
+- ryohack
```

Comment:
Thank you for adding me to the distributor.
But excuse me..., my handle name is "ryo**c**hack".